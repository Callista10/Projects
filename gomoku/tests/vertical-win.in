place B15
place A12
place B14
place A18
place B13
place H2
place B16
place D4
place B17
