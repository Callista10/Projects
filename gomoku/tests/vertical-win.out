Black wins!
B15A12B14A18B13H2B16D4B17
Thank you for playing!
